spec/issues/1061_alternative_time_offsets.spec.js

```diff
@@ -0,0 +1,237 @@
+"use strict";
+
+var Ajv = require("../ajv");
+require("../chai").should();
+
+describe("PR #617, full date format validation should understand leap years", function() {
```

Comment:
Also it is probably copied from another test

Reply:
Yup definitely started by copying to better ensure formatting/code consistency. File's now gone.